tests/python/gaia-ui-tests/gaiatest/apps/contacts/regions/settings_form.py

```diff
@@ -35,18 +36,13 @@ def __init__(self, marionette):
         Base.__init__(self, marionette)
         view = self.marionette.find_element(*self._settings_view_locator)
         Wait(self.marionette).until(lambda m: view.location['y'] == 0)
+        self._order_by_last_name_switch = GaiaBinaryControl(self.marionette, self._order_by_last_name_switch_locator)
 
-    def tap_order_by_last_name(self):
-        last_name = Wait(self.marionette).until(
-            expected.element_present(*self._order_by_last_name_switch_locator))
-        Wait(self.marionette).until(expected.element_displayed(last_name))
-        initial_state = self.is_custom_element_checked(last_name)
-        last_name.tap()
-        self.wait_for_custom_element_checked_state(last_name, checked=not(initial_state))
+    def enable_order_by_last_name(self):
```

Comment:
I didn't want to break the [Law of Demeter](https://en.wikipedia.org/wiki/Law_of_Demeter). To sum up, The law of Demeter says:
```python
my_object1.object2.object3.do_something()
```
is a bad practice, because if one day you change the internal behavior between object2 and object3, this will impact more parts of the code, where as if you just do:
``` python
my_object1.do_something_with_object2()
```

To be honest, I was unsure if we should break this rule too, as most of the time, we just do
```python
def enable_stuff()
    self._stuff.enable()
```

But, sometimes, we have to check another place of the page, [like there](). So I think, for consistency (and the Law of Demeter), it's better to have `enable_stuff()`.